### PR TITLE
Add multiple aggregations on a single column

### LINKED ISSUE #514: Add support for multiple aggregations on a single column
It'd be useful if we could perform multiple aggregations on a single column like pandas.

```python
df.groupby('a').agg({'b': ['min', 'max', 'mean']})
```

### PR DESCRIPTION
**Pull request by harupy**
Resolves #514

### CODE REVIEW

**Comment by harupy** [comment]
It seems doctest failed because of the dict key order issue in Python 3.5. I tried `aggregated[['B', 'C']]` to sort the column order, but it didn't work because of this error (#603 )
```
=================================== FAILURES ===================================
____________ [doctest] databricks.koalas.groupby.GroupBy.aggregate _____________
094 
095         >>> aggregated = df.groupby('A').agg({'B': 'min', 'C': 'sum'})
096         >>> aggregated[['B', 'C']]  # doctest: +NORMALIZE_WHITESPACE
097            B      C
098         A
099         1  1  0.589
100         2  3  0.705
101 
102         >>> aggregated = df.groupby('A').agg({'B': ['min', 'max'], 'C': 'sum'})
103         >>> aggregated  # doctest: +NORMALIZE_WHITESPACE
Differences (unified diff with -expected +actual):
    @@ -1,5 +1,5 @@
    -     B           C
    -   min  max    sum
    -A
    -1    1    2  0.589
    -2    3    4  0.705
    +       C   B
    +     sum min max
    +A
    +1  0.589   1   2
    +2  0.705   3   4
```
https://travis-ci.com/databricks/koalas/jobs/220226148#L1534

File: databricks/koalas/groupby.py [hunk 1]
```diff
@@ -98,26 +99,44 @@ def aggregate(self, func_or_funcs, *args, **kwargs):
         1  1  0.589
         2  3  0.705
 
+        >>> aggregated = df.groupby('A').agg({'B': ['min', 'max'], 'C': 'sum'})
```
Comment (HyukjinKwon) [inline databricks/koalas/groupby.py]: Can we give a shot with `OrderedDict` and see if it works?

Comment (harupy) [reply databricks/koalas/groupby.py]: got it!

Comment (harupy) [reply databricks/koalas/groupby.py]: or we can drop `'C': 'sum'` here because the first doctest already covers it.

Comment (HyukjinKwon) [reply databricks/koalas/groupby.py]: Yea, looks we can do that too.

**Comment by harupy** [comment]
@HyukjinKwon Thank you for the review!

**Comment by ueshin** [comment]
Thanks! merging.

**Comment by harupy** [comment]
@ueshin Thank you!

**Pull request merged**